test/e2e/atlas/alert_settings_test.go

```diff
@@ -137,6 +140,52 @@ func TestAlertConfig(t *testing.T) {
 		}
 	})
 
+	t.Run("Update Setting using file input", func(t *testing.T) {
+		randInit, _ := rand.Int(rand.Reader, big.NewInt(1000))
+		fileName := fmt.Sprintf("%d_alerts.json", randInit.Int64())
```

Comment:
any reason the name of the file needs to be random?

Reply:
We do not need it. I'm overly cautious